### PR TITLE
in OCSMiddleware, also retrieves original error message when response is of type TemplateResponse

### PR DESCRIPTION
**Pull request by StCyr**
Currently, message is only retrieved when response is of type JSONResponse. Why such a difference? I feel offensed 🙃 

Signed-off-by: Cyrille Bollu <cyrpub@bollu.be>

### CODE REVIEW

File: lib/private/AppFramework/Middleware/OCSMiddleware.php [hunk 1]
```diff
@@ -107,6 +108,8 @@ public function afterController($controller, $methodName, Response $response) {
 				if ($response instanceof JSONResponse) {
 					/** @var DataResponse $response */
 					$message = $response->getData()['message'];
+				} elseif ($response instanceof TemplateResponse) {
+					$message = $response->getParams()['message'];
```
Comment (juliushaertl) [inline lib/private/AppFramework/Middleware/OCSMiddleware.php:112]: Why would a template response be returned by an OCS API endpoint? Also the message param is not something that is always present. Maybe you can describe the secenario that caused this a bit further.

**Comment by juliushaertl** [comment]
Ah, and pull requests should go into master first and will then be backported if feasible. :wink:

File: lib/private/AppFramework/Middleware/OCSMiddleware.php (hunk 1 above)
Comment (rullzer) [reply lib/private/AppFramework/Middleware/OCSMiddleware.php:112]: Yes I'd also be interested in this?

Comment (StCyr) [reply lib/private/AppFramework/Middleware/OCSMiddleware.php:112]: Yes, true: I faced the issue when accessing the API directly, by typing the url directly into the browser's location bar.

wich is probably not a common scenario idd

**Comment by StCyr** [comment]
This PR is way out-of-date.

I've just tried to rebase it on master and it seemed quite difficult. 

I propose to close this PR as-is and not merge it.

What do you think @juliushaertl ?

**Comment by juliushaertl** [comment]
Yes, I'd say let's close this, as an OCS API is not supposed to return a TemplateResponse 😉

**Pull request closed without merging**